lms/migrations/versions/0c52a13c6cad_canvas_sso_urls.py

```diff
@@ -0,0 +1,88 @@
+"""
+Upgrade Canvas SSO URLs.
+
+Canvas has upgraded the URLs used in LTI1.3 registrations.
+
+The change is not mandatory or urgent but it should future proof the registrations.
+
+
+From: https://community.canvaslms.com/t5/The-Product-Blog/Minor-LTI-1-3-Changes-New-OIDC-Auth-Endpoint-Support-for/ba-p/551677
+
+URLs that will change:
+
+The OIDC Auth endpoint, also called the Authorization Redirect URL
+Old: https://canvas.instructure.com/api/lti/authorize_redirect
+New: https://sso.canvaslms.com/api/lti/authorize_redirect
+
+The Canvas Public JWKs endpoint
+Old: https://canvas.instructure.com/api/lti/security/jwks
+New: https://sso.canvaslms.com/api/lti/security/jwks
+
+The Grant Host endpoint, sent as the aud claim in LTI Advantage API tokens
+Old: https://canvas.instructure.com/login/oauth2/token
+New: https://sso.canvaslms.com/login/oauth2/token
+
+Revision ID: 0c52a13c6cad
+Revises: 1872d16c28a4
+"""
+import sqlalchemy as sa
+from alembic import op
+
+revision = "0c52a13c6cad"
+down_revision = "1872d16c28a4"
+
+
+def upgrade() -> None:
+    conn = op.get_bind()
+
+    result = conn.execute(
+        """
+        UPDATE lti_registration
+        SET auth_login_url='https://sso.canvaslms.com/api/lti/authorize_redirect'
+        WHERE auth_login_url = 'https://canvas.instructure.com/api/lti/authorize_redirect'"""
```

Comment:
Updating only instances with the known canvas endpoints. Not touching self hosted ones.